modules/monitoring-enabling-a-dedicated-service-monitor.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_content-type: PROCEDURE
+[id="enabling-a-dedicated-service-monitor_{context}"]
+= Enabling a dedicated service monitor
+
+You can enable core platform monitoring to use a dedicated service monitor by configuring the `dedicatedServiceMonitors` key in the `cluster-monitoring-config` `ConfigMap` object in the `openshift-monitoring` namespace.
```

Comment:
```suggestion
You can enable a dedicated service monitor by configuring the `dedicatedServiceMonitors` key in the `cluster-monitoring-config` `ConfigMap` object in the `openshift-monitoring` namespace.
```
Are we enabling core platform monitoring or a dedicated service monitor?

Reply:
Good catch. The word `enable` is confusing here. Changing this to `You can configure core platform monitoring to use`